modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -293,6 +339,8 @@ daemon set "multus" successfully rolled out
 [IMPORTANT]
 ====
 The following scripts reboot all of the nodes in the cluster at the same time. This can cause your cluster to be unstable. Another option is to reboot your nodes manually one at a time. Rebooting nodes one-by-one causes considerable downtime in a cluster with many nodes.
+
+Cluster Operators will not be healthy before rebooting the nodes.
```

Comment:
```suggestion
Cluster Operators will not work correctly before you reboot the nodes.
```
- IBM Style prefers "start" over "boot", but I've retained "boot" here for consistency with the text above this line.
- Avoid giving human characteristics to inanimate objects (e.g., "healthy").

Reply:
good catch, thanks.